samples/snippets/csharp/VS_Snippets_Misc/cds/cs/cds_dictionaryhowto.cs

```diff
@@ -1,114 +1,81 @@
 ﻿//<snippet16>
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
 namespace DictionaryHowTo
 {
-    using System;
-    using System.Collections.Concurrent;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Text;
-    using System.Threading;
-    using System.Threading.Tasks;
-
     // The type of the Value to store in the dictionary.
     class CityInfo : IEqualityComparer<CityInfo>
     {
         public string Name { get; set; }
-        public DateTime LastQueryDate { get; set; }
-        public decimal Longitude { get; set; }
-        public decimal Latitude { get; set; }
-        public int[] RecentHighTemperatures { get; set; }
-
-        public CityInfo(string name, decimal longitude, decimal latitude, int[] temps)
-        {
-            Name = name;
-            LastQueryDate = DateTime.Now;
-            Longitude = longitude;
-            Latitude = latitude;
-            RecentHighTemperatures = temps;
-        }
-
-        public CityInfo()
-        {
-        }
-
-        public CityInfo(string key)
-        {
-            Name = key;
-            // MaxValue means "not initialized".
-            Longitude = Decimal.MaxValue;
-            Latitude = Decimal.MaxValue;
-            LastQueryDate = DateTime.Now;
-            RecentHighTemperatures = new int[] { 0 };
-        }
+        public DateTime LastQueryDate { get; set; } = DateTime.Now;
+        public decimal Longitude { get; set; } = decimal.MaxValue;
+        public decimal Latitude { get; set; } = decimal.MaxValue;
+        public int[] RecentHighTemperatures { get; set; } = new int[] { 0 };
 
         public bool Equals(CityInfo x, CityInfo y)
-        {
-            return x.Name == y.Name && x.Longitude == y.Longitude && x.Latitude == y.Latitude;
-        }
+            => x.Name == y.Name
+            && x.Longitude == y.Longitude
+            && x.Latitude == y.Latitude;
```

Comment:
Totally optional style, but consider:

```suggestion
            => (x.Name, x.Longitude, x.Latitude) == 
                  (y.Name, y.Longitude, y.Latitude);
```